### ISSUE TITLE
Jitsi is shown in English for German users in Safari

### ISSUE THREAD

**Issue by nhoelsken76** [jitsi/jitsi-meet#15301]
### What happened?

In Safari the Jitsi page is shown in English, although user config, as well as browser and system language is set to German.

In Chrome (on Windows machine) it is shown in German if user and Browser locale is set to German.

Expected behaivior:

As in Chrome it is expected to be shown in german in Safari also, if User and Browser is set to German.

### Platform

- [ ] Chrome (or Chromium based)
- [ ] Firefox
- [X] Safari
- [ ] Other desktop browser
- [ ] Android browser
- [ ] iOS browser
- [ ] Electron app
- [ ] Android mobile app
- [ ] iOS mobile app
- [ ] Custom app using a mobile SDK

### Browser / app / sdk version

Safari 18.0 (but other versions also)

### Reproducibility

- [ ] The problem is reproducible on meet.jit.si

### More details?

This is only happening in Safari.

**Comment by saghul** [comment]
Is this reproducible in meet.jit.si?

**Comment by nhoelsken76** [comment]
No. :) it seems to be fixed in the latest Jitsi released. So this issue can be closed. Thank you!

**Comment by saghul** [comment]
Thanks for confirming!
